full-v2.yml

```diff
@@ -882,6 +897,9 @@ paths:
       - in: query
         name: ending_before
         type: string
+      - in: query
+        name: count
+        type: integer
```

Comment:
I noticed in the sis-api PR that we can't use summaries for district admins, but can't we still count them?

Reply:
This is for school admins. We won't be providing counts for district admins 👍

Reply:
Oh actually you're right, count will be supported for district and district admin